internal/ingress/types.go

```diff
@@ -227,7 +227,7 @@ type Location struct {
 	// Backend describes the name of the backend to use.
 	Backend string `json:"backend"`
 	// Service describes the referenced services from the ingress
-	Service *apiv1.Service `json:"service,omitempty" hash:"ignore"`
+	Service *apiv1.Service `json:"-"`
```

Comment:
Is this related to this PR? Why are you changing these?

Reply:
hash:"ignore" is the default tag in hashstructure but we are not using that but json.
Using ignore or - is the same thing (omit the field)